src/__tests__/bundle.spec.ts

```diff
@@ -1619,4 +1619,124 @@ describe('bundleTargetPath()', () => {
       ).not.toThrow();
     });
   });
+
+  describe('root json pointers', () => {
+    it('given an OAS document, should handle them gracefully', () => {
+      const input = {
+        openapi: '3.0.0',
+        paths: {
+          '/users': {
+            $ref: '#',
+          },
+        },
+        components: {
+          schemas: {
+            User: {
+              $ref: '#',
+            },
+          },
+        },
+      };
+
+      const output = bundleTarget({ document: input, path: '#/components', bundleRoot: '#/$defs' });
```

Comment:
What's the relationship between adding `bundleRoot: '#/$defs'` and the root pointer handling problem?

Reply:
This replaces the default value `__bundled__`, but doesn't effect any change on the result.